dnf-docker-test/features/tier-clean.feature

```diff
@@ -0,0 +1,150 @@
+      When I enable repository "ext"
+      Then I successfully run "dnf -y --setopt=keepcache=1 install TestB"
+       And I successfully run "dnf -y --setopt=keepcache=1 upgrade TestA"
+       And I successfully run "find /var/cache/dnf/ -name Test*"
+       And the command stdout should match regexp "ext-.*/packages/TestB-1-1.noarch.rpm" 
+       And the command stdout should match regexp "ext-.*/packages/TestA-2-1.noarch.rpm" 
+      When I successfully run "dnf clean packages"
+      Then I successfully run "find /var/cache/dnf/ -name Test*"
+       And the command stdout should not match regexp "base-.*/packages/TestA-1-1.noarch.rpm" 
+       And the command stdout should not match regexp "ext-.*/packages/TestB-1-1.noarch.rpm" 
+       And the command stdout should not match regexp "ext-.*/packages/TestA-2-1.noarch.rpm" 
+       # metadata are not removed
+       And a file "/var/cache/dnf/base.solv" exists 
+       And a file "/var/cache/dnf/ext.solv" exists 
+       And a file "/var/cache/dnf/base-filenames.solvx" exists 
+       And a file "/var/cache/dnf/ext-filenames.solvx" exists 
+      When I successfully run "find /var/cache/dnf/ -name *.xml"
+      Then the command stdout should match regexp "base-.*/repodata" 
+       And the command stdout should match regexp "ext-.*/repodata" 
+      When I successfully run "find /var/cache/dnf/ -name *.xml.gz"
+      Then the command stdout should match regexp "base-.*/repodata" 
+       And the command stdout should match regexp "ext-.*/repodata" 
+      # cleanup  
+      When I successfully run "dnf -y remove TestA TestB"
+
+  Scenario: Database cache cleanup (dnf clean dbcache)
+  # CACHE-CLEAN-6
+      When I disable repository "ext"
+       And I successfully run "dnf -y --setopt=keepcache=1 install TestA TestC"
+       And I successfully run "dnf clean dbcache"
+      Then a file "/var/cache/dnf/base.solv" does not exist 
+       And a file "/var/cache/dnf/local.solv" does not exist 
+       And a file "/var/cache/dnf/base-filenames.solvx" does not exist
+       And a file "/var/cache/dnf/local-filenames.solvx" does not exist
+      When I successfully run "find /var/cache/dnf/ -name *.xml"
+      Then the command stdout should match regexp "base-.*/repodata" 
+       And the command stdout should match regexp "local-.*/repodata" 
+      When I successfully run "find /var/cache/dnf/ -name *.xml.gz"
+      Then the command stdout should match regexp "base-.*/repodata" 
+       And the command stdout should match regexp "local-.*/repodata" 
+      When I successfully run "find /var/cache/dnf/ -name Test*"
+      Then the command stdout should match regexp "base-.*/packages/TestA-1-1.noarch.rpm" 
+      # cleanup  
+      When I successfully run "dnf -y remove TestA TestC"
+
+  # note: dnf info and repolist do not reflect expire-cache, see bug 1552576
+  Scenario: Expire dnf cache (dnf clean expire-cache) and install previously cached package
```

Comment:
Please, can you temporarily comment out this scenario? We should clear functionality of "dnf clean expire-cache" first.